movie-smt-ui/src/pages/TopRatedMovies/TopRatedMovies.module.scss

```diff
@@ -0,0 +1,14 @@
+.TopRated {
+    color: white;
+    font-size: 0.7em;
```

Comment:
Font size is also something we should store somewhere and keep consistent across the app